test/rules/extensions.spec.ts

```diff
@@ -660,7 +660,7 @@ ruleTester.run('extensions', rule, {
 })
 
 describe('TypeScript', () => {
-  ruleTester.run(`${parsers.TS}: extensions ignore type-only`, rule, {
+  ruleTester.run(`extensions ignore type-only`, rule, {
```

Comment:
Tests with `/` in the rule-name string fail because they are interpreted as namespaces

Reply:
I see!

Let's change the title a little bit to reflect `typescript`.

```suggestion
  ruleTester.run(`typescript - extensions ignore type-only`, rule, {
```

Reply:
FWIW, the `describe()` block also provides that:

```
  TypeScript
    typescript - extensions ignore type-only
      valid
        ✓ import type T from "./typescript-declare"; (4 ms)
        ✓ export type { MyType } from "./typescript-declare"; (1 ms)
      invalid
        ✓ import T from "./typescript-declare"; (1 ms)
        ✓ export { MyType } from "./typescript-declare"; (1 ms)
```